lib/tasks/taxonomy.rake

```diff
@@ -0,0 +1,31 @@
+namespace :taxonomy do
```

Comment:
The `desc` call must be above the `task` call, otherwise `rake -T` will not list the task.